### PR TITLE
Add new line to <br> tag jinja filter

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

**Comment by matt-sm** [comment]
how did this come about?

**Comment by maxious** [comment]
In the email a seller gets back from making a brief response, the line breaks are missing because it's a HTML email.

File: dmutils/filters.py [hunk 1]
```diff
@@ -68,3 +69,14 @@ def timesince(before, now=None, default="just now"):
         return before.diff_for_humans()
 
     return default
+
+
+@evalcontextfilter
+def nl2br(eval_ctx, value):
```
Comment (matt-sm) [inline dmutils/filters.py:75]: feels like a function we should add a test for

**Comment by maxious** [comment]
test added

**Review by matt-sm — APPROVED** [review]

**Pull request merged**